issue-onboarding/onboarding-issue-template.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: triggers.tekton.dev/v1alpha1
```

Comment:
Move this to the dir: https://github.com/AICoE/aicoe-ci/tree/master/triggertemplate